### PR TITLE
Suppress a few checkstyle warnings

### PR DESCRIPTION
**Pull request by snazy**

### CODE REVIEW

File: gc/gc-tool/src/main/java/org/projectnessie/gc/tool/cli/options/MarkOptions.java [hunk 1]
```diff
@@ -36,6 +36,7 @@
 import picocli.CommandLine.ParameterException;
 
 public class MarkOptions {
+  @SuppressWarnings("JavaTimeDefaultTimeZone")
```
Comment (XN137) [inline gc/gc-tool/src/main/java/org/projectnessie/gc/tool/cli/options/MarkOptions.java:39]: nit: maybe add a comment to explain that we are intentionally using the CLI users timezone here (and it will not get mixed with server timezone later on)

File: gc/gc-tool/src/test/java/org/projectnessie/gc/tool/cli/commands/TestMarkOptions.java [hunk 2]
```diff
@@ -148,6 +148,7 @@ public void cutoffPoliciesPerRef() {
         .isEqualTo(CutoffPolicy.numCommits(1));
   }
 
+  @SuppressWarnings("TypeParameterUnusedInFormals")
```
Comment (XN137) [inline gc/gc-tool/src/test/java/org/projectnessie/gc/tool/cli/commands/TestMarkOptions.java:151]: these are errorprone (not checkstyle) warnings:
http://errorprone.info/bugpattern/TypeParameterUnusedInFormals

wondering: couldnt we have fixed this as suggested?

Comment (snazy) [reply gc/gc-tool/src/test/java/org/projectnessie/gc/tool/cli/commands/TestMarkOptions.java:151]: err - yes

File: versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractManyKeys.java [hunk 3]
```diff
@@ -537,7 +537,8 @@ public static Stream<List<String>> progressivelyManyKeyNames() {
         .mapToObj(
             i ->
                 IntStream.range(1, i)
-                    .mapToObj(j -> Long.toString(Math.abs(random.nextLong()), Character.MAX_RADIX))
+                    .mapToObj(
+                        j -> Long.toString(random.nextLong() & Long.MAX_VALUE, Character.MAX_RADIX))
```
Comment (dimas-b) [inline versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractManyKeys.java:541]: Ideally, we need to choose the seed (line 533) again in this case. The changed names are not guaranteed to expose the bug, whose handling that the test is meant to validate.

Comment (XN137) [reply versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractManyKeys.java:541]: sorry i missed reading the context of this change (that we were testing an edge condition with explicitly crafted test data)

Comment (snazy) [reply versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractManyKeys.java:541]: Yea - let's have a separate issue for that.

Comment (dimas-b) [reply versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractManyKeys.java:541]: well, to be fair, the comment was hidden outside of the GH "changed lines" window :)

Comment (dimas-b) [reply versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractManyKeys.java:541]: I ran a quick test with the new name generator and old code base: compared to the old generator the number of failed test cases decreased from 40 to 33 (out of 200). I think the test data is still good enough. I do not think we need to change the seed.
